docs/serving/samples/multi-container/README.md

```diff
@@ -0,0 +1,274 @@
+A simple web app written in Go that you can use for multi container testing.
+
+Follow the steps below to create the sample code and then deploy the app to your
+cluster. You can also download a working copy of the sample, by running the
+following commands:
+
+```shell
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+```
+
+## Before you begin
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+- Make sure multi-container flag is enabled as part of `config-features` configmap.
+
+## Recreating the sample code
+
+Multi container has more than one container, For testing, we will create two containers, the serving container and sidecar container.
+### ServingContainer
+1. `cd knative-docs/docs/serving/samples/multi-container/servingcontainer`
```

Comment:
Explain what this step does, e.g.
1. Navigate to x...

Will this dir exist already or do they need to create it?

Reply:
Thank you
Added a description please do have a look